packages/shared/src/test/helpers.ts

```diff
@@ -2,7 +2,7 @@ import Database from 'better-sqlite3';
 import { BetterSQLite3Database, drizzle } from 'drizzle-orm/better-sqlite3';
 import { migrate } from 'drizzle-orm/better-sqlite3/migrator';
 import tmp from 'tmp';
-import { MockedFunction } from 'vitest';
+import { MockedFunction, beforeAll, beforeEach } from 'vitest';
```

Comment:
I'm not 💯 on the impact of this on the production bundle (will vitest now get bundled?) – I figured we can fix this later if it's a problem

Reply:
Yeah, I think this should be fine.